dtfs-central-api/src/services/state-machines/utilisation-report/event-handlers/report-uploaded/report-uploaded.event-handler.ts

```diff
@@ -33,6 +42,8 @@ const createFacilityUtilisationDataEntityIfNotExists = async (
   return FacilityUtilisationDataEntity.createWithoutUtilisationAndFixedFee({ id: facilityId, reportPeriod, requestSource });
 };
 
+const CHUNK_SIZE_FOR_BATCH_SAVING = 100;
```

Comment:
maybe this can be done in the next PR, but can this be moved to constants?

Reply:
This constant only makes sense in the context of this file and exists purely for readability. Outside the context of this file it would have no meaning, and it is likely that another place which wants to use this would want a different batch size (it depends on many features which are implementation specific like the number of columns, relations etc.), so I don't think we should be adding this to the `constants` directory

Reply:
even if this is only being used once, it should be stored in the constants directory as it is a constant.  It makes it easier to manage/change.  In the future, if we use a shared package for all UKEF constants, it would have to be in there.
PLease can this be moved in the next PR.